jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/DebugHandlerTest.java

```diff
@@ -130,7 +128,15 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     @AfterEach
     public void stopServer() throws Exception
     {
-        server.stop();
+        ArrayByteBufferPool.Tracking byteBufferPool = (ArrayByteBufferPool.Tracking)server.getConnectors()[0].getByteBufferPool();
```

Comment:
Yuck :)
You are only checking one of the tracking buffer pools.
I'd suggest to save them as fields and use the fields here, rather than scavenging them from the server.

Reply:
Done.